downstream/assemblies/hub/assembly-view-only-access.adoc

```diff
@@ -1,5 +1,6 @@
 [id="assembly-view-only-access"]
-= Enabling view-only access for your private automation hub
```

Comment:
Use attribute for private automation hub? (consistent w/ other modules) Not sure if there's still debate about not using attributes in titles due to Git issues.

Reply:
Hi @jself-sudoku. Line 2 was deleted and the new title does include the attribute for Private automation hub.

Reply:
Okay. Thanks, @lmalivert .